app/controllers/locations_controller.rb

```diff
@@ -17,8 +17,10 @@ def external_search
             state: c["address"]["state"] || "",
             county: c["address"]["county"] || "",
             city: c["address"]["city"] || "",
-            lat: c["lat"] || "",
-            lng: c["lng"] || ""
+            # Round coordinates to enhance privacy
+            lat: c["lat"] ? c["lat"].to_f.round(2) : nil,
+            # LocationIQ uses 'lon'
```

Comment:
interesting... I'm not opposed to using `lon` if that is more standard

Reply:
I think you're right that `lng` is more standard actually. based on Google searching of LatLon vs. LatLng.